translations/kr.json

```diff
@@ -1,389 +0,0 @@
-{
```

Comment:
do we want to keep these around in case korea chills out?

Reply:
Made this a separate PR so we can always revert the merge commit in the future